src/app/item-page/simple/item-page.component.html

```diff
@@ -3,6 +3,17 @@
     <div class="item-page" @fadeInOut>
       @if (itemRD?.payload; as item) {
         <div>
+          <ng-container *ngVar="(itemRequest$ | async) as itemRequest">
+            @if (hasValue(itemRequest)) {
+              <div class="alert alert-warning wb-100 mb-2">
+                <p><span role="img" class="request-a-copy-access-icon" [attr.aria-label]="'bitstream-request-a-copy.access-by-token.alt-text' | translate"><i class="fa-solid fa-lock-open"></i></span>{{'bitstream-request-a-copy.access-by-token.warning' | translate}}</p>
+                <!-- Only show the expiry date if it's not null, and doesn't start with the "FOREVER" year -->
+                @if (hasValue(itemRequest.accessExpiry) && !itemRequest.accessExpiry.startsWith('+294276')) {
+                  <p>{{ 'bitstream-request-a-copy.access-by-token.expiry-label' | translate }} {{ itemRequest.accessExpiry }}</p>
+                }
+              </div>
+            }
+          </ng-container>
```

Comment:
Could this be moved to a separate component like the other alerts below?

If we change the REST/resolver logic to still expose expired `RequestItem`s this would also be the place to show an alternative message, e.g. something along the lines of "you're viewing this page with an expired access token, so you no longer have access to certain files"

The date is shown as an ISO timestamp, which is a bit hard to read. Could you transform it with e.g. `| date:"YYYY/MM/dd hh:mm"`?
![20250327-141723](https://github.com/user-attachments/assets/e0555918-3122-4f76-87ef-a4a7811715b3)

Reply:
done